front/app/containers/HomePage/SignedOutHeader/SignedOutHeader.test.tsx

```diff
@@ -21,51 +21,104 @@ jest.mock('hooks/useHomepageSettings', () => {
 });
 
 describe('<SignedOutHeader />', () => {
-  it('renders full_width_banner_layout', () => {
-    mockHomepageSettings.attributes.banner_layout = 'full_width_banner_layout';
+  describe('full_width_banner_layout', () => {
+    beforeEach(() => {
+      render(<SignedOutHeader />);
+    });
 
-    render(<SignedOutHeader />);
-    expect(screen.getByText('Signed out header')).toBeInTheDocument();
-    expect(screen.getByText('Signed out subhead')).toBeInTheDocument();
-    expect(screen.getByTestId('full-width-banner-layout')).toHaveStyle(
-      'width: 100%'
-    );
+    it('renders the headings correctly', () => {
```

Comment:
Would it be possible/useful to make this example shared and reuse it for different layouts?